aws/resource_aws_cloudwatch_log_group.go

```diff
@@ -116,44 +115,42 @@ func resourceAwsCloudWatchLogGroupRead(d *schema.ResourceData, meta interface{})
 	d.Set("arn", lg.Arn)
 	d.Set("name", lg.LogGroupName)
 	d.Set("kms_key_id", lg.KmsKeyId)
+	d.Set("retention_in_days", lg.RetentionInDays)
 
-	if lg.RetentionInDays != nil {
-		d.Set("retention_in_days", lg.RetentionInDays)
+	tags := make(map[string]string, 0)
+	tagsOutput, err := conn.ListTagsLogGroup(&cloudwatchlogs.ListTagsLogGroupInput{
+		LogGroupName: aws.String(d.Id()),
+	})
+	if err != nil {
+		return fmt.Errorf("error listing CloudWatch Logs Group %q tags: %s", d.Id(), err)
 	}
-
-	if !meta.(*AWSClient).IsChinaCloud() && !meta.(*AWSClient).IsGovCloud() {
```

Comment:
since this also affects China - does this fix work in that DC too?

Reply:
I'm not 100% positive since we cannot test AWS China directly (at the moment), but there are some reasons why its likely okay. Its probably important to note that we do not have any access or acceptance testing in place for AWS China and its support is currently best effort based on bug reports. We should probably include this in our documentation/README. 🤔 

It doesn't appear this was ever explicitly requested to be restricted like this as far as I can tell. I did some Git spelunking and I couldn't find an associated reason why AWS China was being included (although probably just for "good measure" when the GovCloud logic was added):
* https://github.com/hashicorp/terraform/commit/55000e4b1ab1db58ac9b05c28aada0528bfec95f
* https://github.com/hashicorp/terraform/commit/028540796dae59b9b5918b64775ba9fa5114b217

I don't see any particular announcement in the AWS China What's New blog for this CloudWatch Log resource tagging, but the GovCloud blog didn't seem to announce it either unless its hidden under another post.

The AWS China CloudWatch documentation points to the regular CloudWatch documentation, which does not list any restrictions in its tagging section.

In the end, I think its worth switching the logic (at the risk of a regression). If the resource tagging is still not supported, we can get a bug report the appropriate error message and can switch it to a more future-proof solution based on that handling rather than depending on a hardcoded conditional that has to be updated in the code.